rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/widget/OneScanCovidRDTBarcodeFactoryTest.java

```diff
@@ -114,12 +111,6 @@ public void testOnActivityResultShouldPopulateCorrectData() throws Exception {
         // verify correct action on back-press
         oneScanCovidRDTBarcodeFactory.onActivityResult(JsonFormConstants.BARCODE_CONSTANTS.BARCODE_REQUEST_CODE, Activity.RESULT_CANCELED, intent);
         Mockito.verify(formFragment).setMoveBackOneStep(true);
-
-        Intent invalidIntent = new Intent();
-        invalidIntent.putExtra(Constants.Config.ENABLE_BATCH_SCAN, true);
-        invalidIntent.putExtra("data", "");
-        oneScanCovidRDTBarcodeFactory.onActivityResult(JsonFormConstants.BARCODE_CONSTANTS.BARCODE_REQUEST_CODE, Activity.RESULT_OK, invalidIntent);
-        oneScanCovidRDTBarcodeFactory.onActivityResult(Activity.RESULT_OK, JsonFormConstants.BARCODE_CONSTANTS.BARCODE_REQUEST_CODE, null);
```

Comment:
Why is this removed?

Reply:
https://github.com/onaio/rdt-standard/pull/656#discussion_r552480001